projects/storefrontlib/src/cms-components/navigation/navigation/navigation-ui.component.ts

```diff
@@ -1,12 +1,56 @@
-import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  Input,
+  OnInit,
+} from '@angular/core';
 import { NavigationNode } from './navigation-node.model';
 
+const COLUMN_SIZE = 10;
+
 @Component({
   selector: 'cx-navigation-ui',
   templateUrl: './navigation-ui.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class NavigationUIComponent {
+export class NavigationUIComponent implements OnInit {
   @Input() dropdownMode = 'list';
   @Input() node: NavigationNode;
+  columns;
+
+  getColumns(children) {
+    const columns = new Array();
```

Comment:
Just use `[]`. I didn't see that anywhere else in our codebase.
```suggestion
    const columns = [];
```